tests/functional/test.h

```diff
@@ -307,18 +313,21 @@ test_parse_args_n(trunk_config           *splinter_cfg,            // OUT
 
    rc = config_parse(master_cfg, num_config, argc, argv);
    if (!SUCCESS(rc)) {
-      return rc;
+      goto out;
```

Comment:
This fixes a minor bug in old code where we return w/o freeing memory allocated for `master_cfg`.